### DISCUSSION TITLE
Solved: Update 14.0.3 not working

### DISCUSSION THREAD

**Discussion by LarsSche** [jomjol/AI-on-the-edge-device#2054] [Q&A] [answered]
Solved:
Copied new SD card files to the SD card and the connection is stable again.
Was able to update firmware to 15.3.

Hi,
last updates I updated the firmware first, reboot and after that I uploaded the whole zip to update everything else.
Worked like charme.
With 14.0.3 I have problems.
firmware.bin update workes without any problems.
But I cannot update the html files now.
I tried:
- Upload the whole update zip
- Upload the html.zip
Both are not working.
In the logs:

0d00h00m09s] 2023-02-21T14:11:52 <WRN> [TFLITE SERVER] Restarted due to an Exception/panic! Postponing first round start by 5 minutes to allow for an OTA Update or to fetch the log!

What can I do?
Thanks

**Comment by caco3** [comment, marked as answer]
just try again until the update is successful. Check the info page to see if both (firmware and Web interface) got updated.
It is normal that it sometimes crashes. Just wait 5 minutes or restart it yourself.

**Reply by LarsSche** [reply]
Restartet a couple of times including cold reboot.
Not working.
Also it's not working zu access the file server. I lose connectivity than, comes back but file server isn't accessible.
Also when I am uploading files for the update it lose connectivity for some pings (~5 pings)
Will check wifi qualitiy.

**Reply by caco3** [reply]
if you can't reach the device reliably, then your wifi connection is not good enough.
Do you use an external antenna?

**Reply by LarsSche** [reply]
Jepp, I am using an external antenna.
And last updates went without any problems.
WIll try it again with an access point in direct neighbourhood

**Reply by caco3** [reply]
Do you have a mash network (multiple Accesspoint with the same Wifi SSID?

Currently this is not working well, meaning it connects to the first one it finds instead of using the one with the strongest signal.

**Reply by LarsSche** [reply]
I have 2 Accesspoints.
For testing I removed one accesspoint.
Also my watermeter is now directly under the remaining accesspoint.
On a bookshelf, approx. 1 meter distance.

But not working.
All other OTA updates workes like charme.
Since 14.03 I have the problems.
I think I will reinstall the device and restore the backup.

**Reply by caco3** [reply]
can you explain bit more what is not working?
And please update to v15.0 and provide the log.

**Reply by LarsSche** [reply]
I wasn't able to update the firmware!
As soon as I uploaded the firmware or the whole update zip the device lost connectivity. I got a message "server interrupted the connection" (or so).
Same happend with access to the file server.

But I solved the problem.
I copied the SD card files from 15.0.3 to the SD card.
Now the connection is stable again, I was able to update the firmware to 15.0.3 and the problems are gone.
It is possible to access the files server again too.
